azure-pipelines.yml

```diff
@@ -199,7 +199,7 @@ jobs:
         displayName: 'Install dependencies'
       - script: npm i webpack@$(webpack_version)
         displayName: 'Install "webpack@$(webpack_version)"'
-      - script: npm run test:coverage -- --ci --reporters="default" --reporters="jest-junit" || $(continue_on_error)
+      - script: npm run test:coverage -- --ci --snapshot-resolver="./test/helpers/snapshotResolver.js" --reporters="default" --reporters="jest-junit" || $(continue_on_error)
```

Comment:
Why we need `--snapshot-resolver` here?

Reply:
It does not work without it when ci flag is used

Reply:
:+1: